### ISSUE TITLE
源码中此处是否出错了？

### ISSUE THREAD

**Issue by 654hhh** [xiehousheng/SemLA#8]
你好，我在阅读代码的过程中发现了以下部分：
        attention = torch.einsum("nls,ncs->nls", attention, feat_sa_ir_flatten)
        attention = torch.sum(attention, dim=2)
想问以下"nls,ncs->nls"这部分是否有错误？
